lib/rucio/transfertool/fts3.py

```diff
@@ -66,6 +66,18 @@
                                   documentation='Number of bulk queries', labelnames=('state', 'host'))
 QUERY_DETAILS_COUNTER = MultiCounter(prom='rucio_transfertool_fts3_query_details', statsd='transfertool.fts3.{host}.query_details.{state}',
                                      documentation='Number of detailed status queries', labelnames=('state', 'host'))
+OVERWRITE_CHECK_COUNTER = MultiCounter(prom='rucio_transfertool_fts3_overwrite_check',
```

Comment:
I would use record_counter() instead of MultiCounter directly. 
In any case, the `statsd` attribute is wrong, because it doesn't have the placeholders for the labels to be filled correctly

Reply:
Something like `transfertool.fts3.overwrite.check.{rsetype}.{rse}.{reason}'

Reply:
Ok. Thanks. Do we have a plan for deprecating statsd? I was hoping I could just say "statsd=None" and not have to worry about it. :-)

I saw some with "record_counter()" but assumed those were older than these. But I'll adjust. Thanks.